### ISSUE TITLE
Ability to prioritize PR to jump to next in line

### ISSUE THREAD

**Issue by chadsmall** [palantir/bulldozer#130]
Is there a mechanism to prioritize a PR so it merges next? If not, this feature would be nice to have.

**Comment by bluekeyes** [comment]
At the moment, there's no priority mechanism in Bulldozer: pull requests are merged in the order events are delivered and processed.

You might be able to simulate this via a second service or script that adds and removes labels to control which PRs are considered for merging, but that would be external to Bulldozer.

Because Bulldozer currently doesn't track any state internally, I think we're in the same position as in #128 (which could be related to this): I'm not sure how to implement something like merge priority without adding additional state tracking.

**Comment by chadsmall** [comment]
Makes sense @bluekeyes. We're considering a bot server that could hook into GitHub events and react accordingly around labels as you suggest. Or as we spike some on Bulldozer, we'll see if any other ideas come to mind w/in the existing implementation and PR if any ideas surface.
-thanks
